.github/workflows/chatops.yml

```
@@ -33,12 +33,20 @@ jobs:
             })
             console.log(pr.data.head.ref)
             return pr.data.head.ref
-            
+      
+      - name: Generate GitHub token
+        id: generate-token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.CHATOPS_APP_ID }}
+          private_key: ${{ secrets.CHATOPS_APP_PRIVATE_KEY }}
+          installation_id: ${{ secrets.CHATOPS_APP_INSTALLATION_ID }}
```

In `v2` we don't have `installation_id`, so maybe we should configure it in below way:

```suggestion
          installation_retrieval_mode: id
          installation_retrieval_payload: ${{ secrets.CHATOPS_APP_INSTALLATION_ID }}          
```

or maybe we should use `installation_retrieval_mode` as `organization` as defined in https://github.com/tibdex/github-app-token/blob/v2.0.0/action.yml ?

---

Good catch